tools/packaging/static-build/cache_components.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../scripts/lib.sh"
+source "${script_dir}/qemu/build-static-qemu-cc.sh"
+
+export KATA_BUILD_CC="${KATA_BUILD_CC:-}"
+
+if [ "${KATA_BUILD_CC}" == "yes" ]; then
+	export qemu_cc_tarball_name="kata-static-qemu-cc.tar.gz"
+else
+	export qemu_cc_tarball_name="kata-static-qemu-cc.tar.gz"
+fi
+
+cache_qemu_artifacts() {
+	local current_qemu_version=$(get_from_kata_deps "assets.hypervisor.qemu.version")
+	create_qemu_cache_asset "${qemu_cc_tarball_name}" "${current_qemu_version}"
+	local qemu_sha=$(calc_qemu_files_sha256sum)
+        echo "${current_qemu_version} ${qemu_sha}" > "latest"
+}
+
+create_qemu_cache_asset() {
+	local component_name="$1"
+	local component_version="$2"
+	local qemu_cc_tarball_path=$(sudo find / -iname "${qemu_cc_tarball_name}")
```

Comment:
Nit: `component_name` == `qemu_cc_tarball_name`. In this function it is using both variables. It should either use one or the other, right?

Reply:
@wainersm I will leave it like this as I am working in other cached components I think I will expand this function to use other component names so I will not change it right now